Starcounter.Uniform/ViewModels/FormItemMetadata.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Starcounter.Uniform.FormItem;
+using Starcounter.Uniform.Generic.FormItem;
+using System;
+using System.Collections.Generic;
+
+namespace Starcounter.Uniform.ViewModels
+{
+    public class FormItemMetadata : Json
+    {
+        internal Dictionary<string, MessageContainer> MessageContainers { get; set; }
```

Comment:
I really dislike internal members - they usually make its declaring class unusable from outside the assembly. Why not make it private and initialize in `Init` (because constructors don't work)?